core/new-gui/package-lock.json

```diff
@@ -4566,6 +4566,16 @@
       "integrity": "sha512-jDctJ/IVQbZoJykoeHbhXpOlNBqGNcwXJKJog42E5HDPUwQTSdjCHdihjj0DlnheQ7blbT6dHOafNAiS8ooQKA==",
```

Comment:
Please remove changes on this file since `package.json` is not changed.

Reply:
I will remove this.